packages/2d/src/lib/components/Media.ts

```diff
@@ -124,12 +141,47 @@ export abstract class Media extends Asset {
         `volumes cannot be negative - the value will be clamped to 0.`,
       );
     }
+    const media = this.mediaElement();
+    media.volume = Math.min(Math.max(volume, 0), 1);
+
     if (volume > 1) {
+      if (this.allowVolumeAmplificationInPreview()) {
+        this.amplify(media, volume);
+        return;
+      }
       console.warn(
-        `the browser does not natively support volumes higher than 1 - your preview will have a volume of 1, but your exported video will have a volume of ${volume}`,
+        `you have set the volume of node ${this.key} to ${volume} - your video will be exported with the correct volume, but the browser does not support volumes higher than 1 by default. To enable volume amplification in the preview, set the "allowVolumeAmplificationInPreview" of your <Video/> or <Audio/> tag to true. Note that amplification for previews will not work if you use autoplay within the player due to browser autoplay policies: https://developer.chrome.com/blog/autoplay/#webaudio.`,
       );
     }
-    this.mediaElement().volume = Math.min(Math.max(volume, 0), 1);
+  }
+
+  @computed()
+  protected amplify(node: HTMLMediaElement, volume: number) {
```

Comment:
You could make it more readable like this (untested)
```ts
@computed()
protected amplify(node: HTMLMediaElement, volume: number) {
  const key = `${viaProxy(this.fullSource())}/${this.key}`;

  // Amplification already exists for this key
  if (Media.amplificationPool[key]) {
    Media.amplificationPool[key].gainNode.gain.value = volume;
    return;
  }

  const audioContext = new AudioContext();
  const sourceNode = audioContext.createMediaElementSource(node);
  const gainNode = audioContext.createGain();

  gainNode.gain.value = volume;
  sourceNode.connect(gainNode);
  gainNode.connect(audioContext.destination);

  Media.amplificationPool[key] = {audioContext, sourceNode, gainNode};

  // Audio context does not need to be resumed or if we're not in a browser environment
  if (typeof window === 'undefined' || audioContext.state !== 'suspended') {
    return;
  }

  // Start audio context after user interaction, necessary due to browser autoplay policies
  const handleInteraction = () => {
    Media.amplificationPool[key].audioContext.resume();
    window.removeEventListener('click', handleInteraction);
  };
  window.addEventListener('click', handleInteraction);
}
```

Reply:
done